buildroot/share/PlatformIO/scripts/common-dependencies.py

```diff
@@ -132,8 +132,7 @@ def force_ignore_unused_libs():
 	known_libs = get_all_known_libs()
 	diff = (list(set(known_libs) - set(env_libs)))
 	lib_ignore = env.GetProjectOption('lib_ignore') + diff
-	if verbose:
-		print("Ignore libraries:", lib_ignore)
+	blab(f'Ignore libraries: {lib_ignore}')
```

Comment:
Please note this syntax breaks python 2.7 support. Maybe a python version check should be done before ? to avoid weird errors later...

Reply:
the problem is not i dont use python3 but ubuntu 18.04 and fedora 29 (at least) symlinks "python" to python2 by default